packages/patternfly-4/react-core/src/components/AboutModal/helpers/AboutModalContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { AboutModalBoxContent } from './AboutModalBoxContent';
+import { AboutModalBoxHeader } from './AboutModalBoxHeader';
+import { AboutModalBoxHero } from './AboutModalBoxHero';
+import { AboutModalBoxBrand } from './AboutModalBoxBrand';
+import { AboutModalBoxCloseButton } from './AboutModalBoxCloseButton';
+import { AboutModalBox } from './AboutModalBox';
+import Backdrop from '../../Backdrop/Backdrop';
+import Bullseye from '../../../layouts/Bullseye/Bullseye';
+
+export interface AboutModalContainerProps extends React.HTMLProps<HTMLDivElement> {
+  /** content rendered inside the About Modal Box Content.  */
+  children: React.ReactNode;
+  /** additional classes added to the About Modal Box  */
+  className?: string;
+  /** Flag to show the About Modal  */
+  isOpen?: boolean;
+  /** A callback for when the close button is clicked  */
+  onClose?: () => void;
+  /** Product Name  */
+  productName?: string;
+  /** Trademark information  */
+  trademark?: string;
```

Comment:
Is there a reason this is no longer required?

Reply:
Yes. I removed it to be consistent with `AboutModal` (the component consumers use), which does not require it. Otherwise a type error gets thrown in `AboutModal.tsx` when it constructs an `AboutModalContainer`.

Reply:
Nice catch.